### ISSUE TITLE
Broken dialogue randomizing for bots

### ISSUE THREAD

**Issue by brooklynzsmac** [Sh0wdown/UnofficialCrusaderPatch#249]
A few of dialogue lines from every AI render to be unused, these are *_extra_01, *_noattack_01, *_nohelp_01. Four enemies from Stronghold 1 have even more of these: 
pg_anger_01 
pg_anger_03
pg_plead_02
pg_taunt_05
pg_taunt_08
pg_vict_01
pg_vict_04
rt_anger_02
rt_anger_03
rt_plead_02
rt_taunt_03
rt_taunt_04
rt_taunt_06
rt_taunt_07
rt_vict_01
sn_anger_02
sn_plead_02
sn_taunt_02
sn_taunt_06
sn_taunt_08
sn_vict_01
sn_vict_02
wf_anger_01
wf_anger_03
wf_plead_02
wf_taunt_03
wf_taunt_06
wf_taunt_07
wf_taunt_08
wf_vict_03
wf_vict_04

These all dialogues remain unused. I wonder if something is wrong with a dialogue randomizer. Anyway, can those be anyhow restored?

**Comment by LordDerMoff** [comment]
A lot of changes made by the mod are controversial. For this reason it is important to be able to activate them one by one. THIS change just can't be controversial! It is gamechanging, but not gamplaye wise but in matter of atmosphere. It doesn't change the Atmosphere, it  just adds a lot of stuff i bet everyone would die to hear during a game. If there might be a way (i know you will find one, you always did) this would be amazing at very least

**Comment by GutentagderHerr** [comment]
> 
> 
> A few of dialogue lines from every AI render to be unused, these are *_extra_01, *_noattack_01, *_nohelp_01. Four enemies from Stronghold 1 have even more of these:


The _Nohelp_01 or _noattack_01 is used, if you ask him to attack or to defend him, during he needs help himself.

**Comment by brooklynzsmac** [comment]
No, they arent, *_nohelp_02 and *_noattack_02 are always used instead.

**Comment by PitchNeeded** [comment]
Some AIs have lines they are supposed to use (if they are your ally) when they are about to siege an enemy castle, but these aren't in the game.

**Comment by brooklynzsmac** [comment]
Which lines exactly? @PitchNeeded
